schedule/yast/autoyast_btrfs.yaml

```diff
@@ -0,0 +1,57 @@
+name:           autoyast_btrfs
+description:    >
+  Non-default btrfs subvolume structure test for autoyast installation.
+  Verify btrfs in new installation and generation of autoinst.xml.
+  Same as autoyast_btrfs, but with product and and addons defined for SLE15.
+vars:
+  AUTOYAST: autoyast_sle15/autoyast_btrfs.xml
+  AUTOYAST_PREPARE_PROFILE: 1
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/wicked
+  - autoyast/repos
+  - autoyast/clone
+  - autoyast/logs
+  - autoyast/autoyast_reboot
+  - installation/grub_test
+  - installation/first_boot
+  - console/system_prepare
+  - autoyast/verify_btrfs
+  - autoyast/verify_cloned_profile
+test_data:   
+  profile:
+    partitioning:
+      - drive:
+          unique_key: enable_snapshots
```

Comment:
Could you please document this special key somewhere, so we know how to use it next time?

Reply:
done